plugin.audio.eco99music/addon.py

```diff
@@ -0,0 +1,143 @@
+# coding=utf-8
+
+import sys
+import os
+import urllib
+import urlparse
+import xbmcaddon
+import xbmcgui
+import xbmcplugin
+import requests
+from bs4 import BeautifulSoup
+import re
+
+
+def build_url(query):
```

Comment:
This is a minor comment and you are not required to change it. Normally it is a bad practice to use methods with only one or two lines of code